chroma-manager/tests/integration/utils/test_blockdevices/test_blockdevice_linux.py

```diff
@@ -20,6 +20,10 @@ def preferred_fstype(self):
     def device_path(self):
         return self._device_path
 
+    @property
+    def wipe_device_commands(self):
+        return ['wipefs -a {}'.format(self.device_path)]
+
```

Comment:
Is this needed here when it's already defined (identically) [above](https://github.com/intel-hpdd/intel-manager-for-lustre/pull/587/files#diff-75dc17464a609d28560da4a3961fc46cR54)?